provider/cluster_rosa_classic_resource_test.go

```diff
@@ -245,6 +248,8 @@ var _ = Describe("Rosa Classic Sts cluster", func() {
 			Expect(clusterState.CloudRegion.Value).To(Equal(regionId))
 			Expect(clusterState.MultiAZ.Value).To(Equal(multiAz))
 			Expect(clusterState.Properties.Elems["rosa_creator_arn"].Equal(types.String{Value: rosaCreatorArn})).To(Equal(true))
+			Expect(clusterState.Properties.Elems["rosa_tf_version"].Equal(types.String{Value: build.Version})).To(Equal(true))
```

Comment:
Please add it also in the subsystem test - check that the resource holds those properties `Expect(resource).To(MatchJQ(attributes.properties....) ) `